src/site/resources/styleguides/google-java-style-20220203/javaguide.html

```
@@ -1090,18 +1234,6 @@ <h3 id="s6.2-caught-exceptions">6.2 Caught exceptions: not ignored</h3>
 return handleTextResponse(response);
 </pre>
 
-<p class="exception"><strong>Exception:</strong> In tests, a caught exception may be ignored
-without comment <em>if</em> its name is or begins with <code class="prettyprint lang-java">expected</code>. The
-following is a very common idiom for ensuring that the code under test <em>does</em> throw an
-exception of the expected type, so a comment is unnecessary here.</p>
-
-<pre class="prettyprint lang-java">try {
-  emptyStack.pop();
-  fail();
-} catch (NoSuchElementException expected) {
-}
-</pre>
-
```

this is functional update  for us.
allowance to have special name for exception as excuse is removed.
https://github.com/checkstyle/checkstyle/blob/56b12bed92a807f90e21cc136c7f6afdc9614ae8/src/main/resources/google_checks.xml#L434-L436

we need to remove this property.
And we need to find a way to demand some comment if exception is swallowed.
@mohitsatr , please investigate. It might need separate issue as update will not be simple.

---

@mohitsatr , please do

---

@romani
https://google.github.io/styleguide/javaguide.html#s6.2-caught-exceptions
> When it truly is appropriate to take no action whatsoever in a catch block, the reason this is justified is explained in a comment.

is it really possibe to check this requirements? 
swallowing exception through comment is just another way to allow users not take any meaningful action on exceptions?